eng/jacoco-test-coverage/pom.xml

```diff
@@ -205,7 +205,7 @@
     <!--    <dependency>-->
     <!--      <groupId>com.azure</groupId>-->
     <!--      <artifactId>azure-storage-blob-nio</artifactId>-->
-    <!--      <version>12.0.0-beta.1</version> &lt;!&ndash; {x-version-update;com.azure:azure-storage-blob-nio;current} &ndash;&gt;-->
+    <!--      <version>12.0.0-beta.2</version> &lt;!&ndash; {x-version-update;com.azure:azure-storage-blob-nio;current} &ndash;&gt;-->
```

Comment:
@rickle-msft Reading the comment above this dependency, it seems the conditions have been met to uncomment this and start collecting code coverage for Jacoco. I'm going to approve this PR but I expect either another PR or another commit that uncomments this.